Source/Core/VideoCommon/DriverDetails.h

```diff
@@ -97,10 +97,10 @@ namespace DriverDetails
 		// Started Version: ?
 		// Ended Version: 13.9 working for me (neobrain).
 		// Affected OS: Linux
-		// Pinned memory is disabled for index buffer as the amd driver (the only one with pinned memory support) seems
+		// Pinned memory is disabled for index buffer as the AMD driver (the only one with pinned memory support) seems
 		// to be broken. We just get flickering/black rendering when using pinned memory here -- degasus - 2013/08/20
 		// This bug only happens when paired with base_vertex.
-		// Please see issue #6105 on google code. Let's hope buffer storage solves this issues.
+		// Please see issue #6105 on Google Code. Let's hope buffer storage solves this issues.
```

Comment:
It's just a note saying what issue it was reported in for the developer if they want more information.
The few lines above state which video driver version fixed the issue.